README.md

```diff
@@ -281,7 +281,7 @@ defined in the `Rectangle` class is called.
 Press "Step Into" to calculate and return the area to the `main` method.
 
 Keep pressing "Step Into" to loop through the rest of the elements in the array, calling `getArea()` and
-`getCircumference()` on each shape.  Confirm when the shape is a `Circle` object,
+`getPerimeter()` on each shape.  Confirm when the shape is a `Circle` object,
 the `getArea()` in `Circle` is executed:
 
 ![loop step 3](https://curriculum-content.s3.amazonaws.com/6677/pillars/interface_step3.png)
```

Comment:
Unfortunately, these screenshots here would need to be updated too

Reply:
I did update them but it looks like the amazonaws cache took a while to update. they should be ok now.